eox_nelp/edxapp_wrapper/backends/certificates_m_v1.py

```diff
@@ -12,4 +13,7 @@ def get_generated_certificates_admin():
     Returns:
         GeneratedCertificateAdmin class.
     """
-    return admin.GeneratedCertificateAdmin
+    if apps.is_installed(certificates.models.__package__):
+        return certificates.admin.GeneratedCertificateAdmin
+
+    return object
```

Comment:
I think is better this one `admin.ModelAdmin` but I didn't consider that, anyway that won't change anything since the register function won't register that

Reply:
I amended that to the unique commit